packages/discord.js/src/structures/StringSelectMenuBuilder.js

```diff
@@ -74,5 +74,5 @@ module.exports = StringSelectMenuBuilder;
 
 /**
  * @external BuildersSelectMenu
- * @see {@link https://discord.js.org/#/docs/builders/main/class/SelectMenuBuilder}
+ * @see {@link https://discord.js.org/docs/packages/builders/main/SelectMenuBuilder:Class}
```

Comment:
This doesn't exist, I think?

Reply:
It indeed doesn't, I guess this should be a `StringSelectMenuBuilder`? Did it ever exist in the original docs though?

Reply:
No, it didn't exist in the older one. The reason you see it there is because the documentation stopped generating for subpackages a long time ago.

Reply:
Should we use the [`BaseSelectMenuBuilder` class](https://discord.js.org/docs/packages/builders/main/BaseSelectMenuBuilder:Class) instead then?

Reply:
No, StringSelectMenuBuilder, that is the actual replacement for it